readthedocs/builds/forms.py

```diff
@@ -33,3 +39,12 @@ def save(self, *args, **kwargs):
         obj = super(VersionForm, self).save(*args, **kwargs)
         if obj.active and not obj.built and not obj.uploaded:
             trigger_build(project=obj.project, version=obj)
+
+    def clean(self):
+        cleaned_data = super(VersionForm, self).clean()
+        if self.instance.pk is not None:  # new instance only
```

Comment:
The comment is wrong or misleading, this will actually only act on "old" instances in the sense of ones that already existed before the form was saved.